server/model/monitor.js

```diff
@@ -1379,11 +1380,12 @@ class Monitor extends BeanModel {
                 return;
             }
 
-            let notifyDays = await setting("tlsExpiryNotifyDays");
+            let notifyDays = await Settings.get("tlsExpiryNotifyDays");
             if (notifyDays == null || !Array.isArray(notifyDays)) {
                 // Reset Default
-                await setSetting("tlsExpiryNotifyDays", [ 7, 14, 21 ], "general");
+                await Settings.set("tlsExpiryNotifyDays", [ 7, 14, 21 ], "general");
                 notifyDays = [ 7, 14, 21 ];
+                await Settings.set("tlsExpiryNotifyDays", notifyDays, "general");
```

Comment:
This line is not presented in the old code.

Reply:
I suppose that would be a question for @CommanderStorm in #4391: https://github.com/louislam/uptime-kuma/pull/4391/files#diff-0c4874f149ad01732ad730f36c95c99e37f5c3f13e9b52f628e2e284daa72d27R1388

Reply:
![image](https://github.com/user-attachments/assets/a28fcdb6-093c-4c13-b9cf-1929bc18b15b)

Yea, the first time of setting the setting should not exist.

Reply:
Fixed: https://github.com/SapphicCode/fork-uptime-kuma/commit/861e100d82a8ed4aaa860259dae0f6d97549213a